src/doc/man/cargo-fix.md

```diff
@@ -52,15 +52,37 @@ The `cargo fix` subcommand can also be used to migrate a package from one
    (without the `--edition` flag) to apply any suggestions given by the
    compiler.
 
-And hopefully that's it! Just keep in mind of the caveats mentioned above that
-`cargo fix` cannot update code for inactive features or `cfg` expressions.
-Also, in some rare cases the compiler is unable to automatically migrate all
-code to the new edition, and this may require manual changes after building
-with the new edition.
+If you encounter failing fixes, read the troubleshooting section.
 
 [edition]: https://doc.rust-lang.org/edition-guide/editions/transitioning-an-existing-project-to-a-new-edition.html
 [edition field]: ../reference/manifest.html#the-edition-field
 
+### Troubleshooting failing fixes
+
+While `cargo fix` strives to automatically apply fixes, it can still fail for
+various reasons. Here are some suggestions for troubleshooting fix failures:
+
+* Unlike other commands, `cargo fix` implies `--all-targets` by default.
+  Try fixing one Cargo target at a time instead of everything in one run.
+  For instance, narrow the scope to a single binary target:
+
+  ```sh
+  cargo fix --bin mybin
+  ```
+
+* If you have a Cargo workspace containing multiple packages,
+  fix them one by one to avoid unwanted feature unification between packages.
+  For instance, fix only the `bar` package :
+
+  ```sh
+  cargo fix --package bar
+  ```
```

Comment:
imo these give suggestions without context around them or why other failures may happen which

This also feels like its heavily written for editions but is at the same heading level as editions

Reply:
Yes it is missing contexts. I am not sure how to put one. The biggest one in my mind is feature unifications. There might be some other I am not aware of.

>  feels like its heavily written for edition

Not really? One can run plain `cargo fix` and hitting it hard due to having mutually exclusive features.

Reply:
Hmm, maybe I can reframe as edition  troubleshooting, as we do give users specific steps for edition migration

Reply:
Moved it to under migration and polished it a bit.